helix-view/src/document.rs

```diff
@@ -89,7 +89,7 @@ pub struct Document {
 
     syntax: Option<Syntax>,
     // /// Corresponding language scope name. Usually `source.<lang>`.
-    pub(crate) language: Option<Arc<LanguageConfiguration>>,
+    pub language: Option<Arc<LanguageConfiguration>>,
```

Comment:
Don't expose this, call `doc.language_config()` instead.  We want to avoid being able to directly mutate most of the doc fields since there's more invariants.